contracts/ethereum/contracts/VEGovLockup.sol

```diff
@@ -0,0 +1,181 @@
+// SPDX-License-Identifier: GPL
+
+pragma solidity 0.8.11;
+pragma abicoder v2;
+
+import "./BaseNativeToken.sol";
+import "./GovToken.sol";
+
+import "./IERC20.sol";
+import "./IEmergencyMode.sol";
+
+import "./LibGovernanceCalc.sol";
+
+/// @dev default maxLockTime to use as the max amount that could be
+///      locked up
+uint256 constant MAX_LOCKUP_TIME = 365 days;
+
+struct Lockup {
+    /// @dev lockLength is the number of seconds the amount was locked up for
+    uint256 lockLength;
+
+    /// @dev fluid/fweth BPT token we're locking up
+    uint256 bptAtLock;
+
+    /// @dev lockTime is the timestamp the token was locked up at
+    uint256 lockTime;
+}
+
+contract VEGovLockup is IEmergencyMode {
```

Comment:
I'd argue "no" since we can redeploy a fixed contract with the power to drain the original investment from the other VE lockup and have it have similar power with the DAO